mfjcs-rest/src/main/java/org/mfjcs/core/ItemStoreImpl.java

```diff
@@ -0,0 +1,55 @@
+package org.mfjcs.core;
+
+import static org.joda.time.format.ISODateTimeFormat.basicDate;
+import static org.joda.time.format.ISODateTimeFormat.basicDateTime;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.mfjcs.api.Item;
+import org.mfjcs.api.ItemStore;
+import org.mfjcs.api.ItemStoreException;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.dropwizard.jackson.Jackson;
+
+public class ItemStoreImpl implements ItemStore {
```

Comment:
and the sharing/planning of work
suggestion: chatter on slack some more ... but also gradually round up in some docs/diagrams?